Cubical/Structures/Option.agda

```diff
@@ -0,0 +1,92 @@
+{-# OPTIONS --cubical --no-import-sorts --no-exact-split --safe #-}
```

Comment:
Should we really keep this file now that we have Maybe?

Reply:
No, I just forgot to remove it